src/components/accessories/patientPicker/PatientPicker.tsx

```diff
@@ -300,14 +300,39 @@ const PatientPicker: FC<IProps> = ({
                     errorText={getErrorText("birthDate")}
                     label={t("patient.birthdate")}
                     onChange={dateFieldHandleOnChange("birthDate")}
+                    disabled={searchStatus === "LOADING"}
                   />
                 </div>
                 <div className="patientSearchForm__item submit_button">
-                  <Button type="submit">{t("common.search")}</Button>
+                  <Button
+                    type="submit"
+                    variant="contained"
+                    disabled={searchStatus === "LOADING"}
+                  >
+                    {t("common.search")}
+                  </Button>
                 </div>
+                {searchStatus === "LOADING" && (
+                  <CircularProgress
+                    style={{ marginLeft: "50%", position: "absolute" }}
+                  />
+                )}
               </div>
             </form>
-            <div className="patientSearchResult">{renderSearchResults()}</div>
+            <div className="patientSearchResult">
+              {renderSearchResults()}
+              {(patientData?.length ?? 0) > 0 && (
```

Comment:
I would add a searchStatus === SUCCESS, don't you?

Reply:
Ok right